pkg/mcp/creds/create.go

```diff
@@ -0,0 +1,53 @@
+//  Copyright 2018 Istio Authors
```

Comment:
Please move watcher and this file under /pkg/mtls ( or similar name ) - ads client and server will need to use it as well ( we don't have rotation yet in pilot - that's why we still use envoy in front )